apps/admin_api/test/admin_api/v1/controllers/admin_auth/account_membership_controller_test.exs

```diff
@@ -257,7 +257,7 @@ defmodule AdminAPI.V1.AdminAuth.AccountMembershipControllerTest do
       assert response["data"]["code"] == "client:invalid_parameter"
 
       assert response["data"]["description"] ==
-               "The `redirect_url` is not allowed to be used. Got: '#{redirect_url}'."
+               "The given `redirect_url` is not allowed to be used. Got: '#{redirect_url}'."
```

Comment:
Can we change this error to -> "The given `redirect_url` is not allowed. Got: '#{redirect_url}'." ?